docs/INTEGRATED_VECTORIZATION.md

```diff
@@ -0,0 +1,14 @@
+[Back to *Chat with your data* README](../README.md)
+
+# Integrated Vectorization
+[**USER STORY**](#user-story) | [**DEPLOYMENT INSTRUCTIONS**](#local-deployment-instructions)
+\
+\
+![User Story](/media/userStory.png)
+## User Story
+This feature allows chunking and vectorization of data during ingestion into Azure AI Search through built-in pull-indexers. It supports automatic processing of data directly from storage - meaning the user can just upload their data to Azure Blob Storage and the built-in pull-indexers will do the chunking, vectorization and indexing. This removes the need for Chat With Your Data to explicitly perform chunking, vectorization and pushing to the search index.
```

Comment:
Can we add a link to https://learn.microsoft.com/en-us/azure/search/vector-search-integrated-vectorization